### PR TITLE
Add supplystack/codacy-coverage-reporter Plugin

### PR DESCRIPTION
**Pull request by BitProcessor**

### CODE REVIEW

File: content/SupplyStack/codacy-coverage-reporter/index.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+---
+date: 2019-11-18T00:00:00+00:00
+title: Codacy Coverage Reporter
+author: SupplyStack
+tags: [ coverage ]
+logo: codacy.svg
+repo: supplystack/codacy-coverage-reporter
+image: supplystack/codacy-coverage-reporter:6.0.6
```
Comment (appleboy) [inline content/SupplyStack/codacy-coverage-reporter/index.md]: remove `6.0.6` version?

Comment (BitProcessor) [reply content/SupplyStack/codacy-coverage-reporter/index.md]: done

File: content/SupplyStack/codacy-coverage-reporter/index.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+Checkout [Docker Hub](https://hub.docker.com/r/supplystack/codacy-coverage-reporter/tags) for all versions of the plugin. Tag follows the Codacy Coverage Reporter version inside the image.
+
+
+```yaml
+kind: pipeline
+name: default
+
+steps:
+- name: codacy
+  image: supplystack/codacy-coverage-reporter:6.0.6
```
Comment (BitProcessor) [inline content/SupplyStack/codacy-coverage-reporter/index.md]: Also removed version here

**Comment by appleboy** [comment]
Need another maintainer approval @tboerger

**Comment by tboerger** [comment]
Oh, deleted the repository?

**Comment by BitProcessor** [comment]
Hi Thomas,

We ditched Codacy in favour of Sonar. So no further maintenance to the plugin will be performed.
That’s why I deleted the repo and PR.

Best regards,
Maarten

**Pull request closed without merging**